src/structures/DateTimeZoneId.php

```diff
@@ -66,11 +66,20 @@ public function nanoseconds(): int
 
     /**
      * identifier for a specific time zone
-     * @return int
+     * @return string
      */
-    public function tz_id(): int
+    public function tz_id(): string
     {
         return $this->tz_id;
     }
 
+    public function __toString(): string
+    {
+        $dt = \DateTime::createFromFormat('U', $this->seconds, new \DateTimeZone('UTC'));
+        $fraction = new \DateInterval('PT0S');
+        $fraction->f = $this->nanoseconds / 1000000000;
+        $dt->add($fraction);
+        $dt->setTimezone(new \DateTimeZone($this->tz_id));
+        return $dt->format('Y-m-d\TH:i:s.uP');
```

Comment:
Looks good! I would return the date time like this:

```php
$dt->format(DATE_ISO8601)
```

This way we are sure the format will be accepted in all major languages and frameworks

Reply:
I was comparing with response from http api and this was format there.

Reply:
In that case, it's perfect!